examples/server/README.md

```diff
@@ -556,9 +556,51 @@ Run with bash:
 bash chat.sh
 ```
 
-### API like OAI
+### API like OpenAI
 
-The HTTP server supports OAI-like API
+The HTTP server supports OpenAI-like API
+
+### API errors
+
+Server returns error in the same format as OpenAI API: https://platform.openai.com/docs/guides/error-codes/api-errors
```

Comment:
@ggerganov I am wondering if we should avoid too much reference to the OAI documentation, looks like a copyright issue, just saying...  although it's just about errors

Reply:
I'm not lawyer but I know that there was a lawsuit between Google and Oracle a long time ago, where Oracle suit Google of using Java APIs in Android. Oracle lost the case because programming interfaces in general cannot be copyrighted.

However using the name "OpenAI" maybe problematic, I don't know if we need to specifically state in the docs that "We are not sponsored or having any partnership with OpenAI". Their Branding guideline does somewhat mention about this: https://openai.com/brand (Section: Non-partnerships)

Reply:
you can write OpenAI compatible yes, but putting a link to their specifications is problematic

Reply:
maybe as I did on the main readme.md, you can link to their github OAS.

Reply:
Yeah that's reasonable, I'll remove the link then.